oe_corporate_blocks.post_update.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Corporate blocks post updates.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Delete the config for Site Switcher.
+ */
+function oe_corporate_blocks_post_update_00001() {
```

Comment:
We have just realized that we will probably need to account for the major version change in the post update naming, something like `hook_post_update_20001`, since in this way we will be sure that 2.x update hooks will be always ran after 0.x or 1.x ones. We will have to git it some thoughts.